composite/action.yml

```diff
@@ -0,0 +1,50 @@
+# action.yml
+name: 'sarif-to-comment-action'
+author: tomwillis608
+description: 'Run @security-alert/sarif-to-comment to post the content of a SARIF file to a GitHub pull request.'
+inputs:
+  sarif-file:
+    description: 'Path to SARIF file to add to pull request comment'
+    required: true
+  token:
+    description: 'GitHub Access Token'
+    required: true
+  repository:
+    description: 'GitHub repository as org/repo.'
+    required: true
+  branch:
+    description: 'The branch for the pull request.'
+    required: true
+  pr-number:
+    description: 'The number for the pull request.'
+    required: true
+  title:
+    description: 'The title of the issue'
+    default: 'SARIF vulnerabilities report'
+    required: false
+  show-rule-details:
+    description: 'Flag to show or hide rule details.'
+    default: 'true'
+    required: false
+  dry-run:
+    description: 'Add "true" flag here for dry run mode. For testing.'
+    default: 'false'
+    required: false
+  odc-sarif:
+    description: 'SARIF file is in OWASP Dependency Check dialect.'
+    default: 'false'
+    required: false
+outputs:
+  output:
+    description: 'The output of the docker run.'
+runs:
+  using: 'composite'
+  steps:
+    - name: Download action
+      run: npm install @security-alert/sarif-to-comment@1.10.10
```

Comment:
The [Dockerfile](https://github.com/sett-and-hive/sarif-to-comment-action/blob/main/Dockerfile#L8-L10) has the version of `@security-alert/sarif-to-comment` pinned to 1.10.10 (which is the version mentioned here), so this makes sense.

That said, if a vulnerability is discovered in 1.10.10, changing the repo to use an updated version would involve looking in multiple places (through no fault of the PR author!).

Suppose, instead, a `package.json` file is added to include that package.  A tool such as Dependabot or Renovate could be used to maintain that dependency while minimizing the drudgery of a human being needing to `grep` around to find that dependency's pinned version.

Reply:
This is the prompt one may be inclined to respond to with, "we accept PRs"..  Acknowledged.

Also, this isn't a "problem" with this PR and it doesn't require any changes from its current form.  I'm just adding these notes as single comments.  They're just thoughts, not required changes.

@matejdro did a very good job and nothing I wrote here is meant to imply anything negative or demonstrate any lack of appreciation.